util/async.c

```diff
@@ -144,12 +144,12 @@ void aio_bh_call(QEMUBH *bh)
 /* Multiple occurrences of aio_bh_poll cannot be called concurrently. */
 int aio_bh_poll(AioContext *ctx)
 {
-    BHListSlice slice;
+    BHListSlice *slice = g_malloc(sizeof(BHListSlice));
```

Comment:
Why this? Please see also the mainstream QEMU of newer versions for a better fix. Using it would make updating QEMU base easier.